src/library/onlineStatusContext.js

```diff
@@ -17,24 +17,25 @@ const OnlineStatusProvider = ({ children, value }) => {
 
   const rePingApiRef = useRef()
 
-  const stopPingingApi = useCallback(() => {
-    setIsServerReachable(null)
-    clearInterval(rePingApiRef.current)
-  }, [])
+  const pingApi = () => {
+    axios
+      .get(`${apiBaseUrl}/health/`, {
+        cache: false,
+        method: 'HEAD',
+        timeout: 2000,
```

Comment:
timeout here

Reply:
moved this outside of the useEffect so we can manually call it when the user toggles online.

Reply:
Julia, I removed the timeout based on Kim's comment about slow 3g and it giving false fails.